lib/queries.rb

```diff
@@ -0,0 +1,275 @@
+class Queries
+  def self.default
+    queries = {
```

Comment:
quries{
  :common_likers = {
      id: -1, name: 'common likers', description: ''
   }
}

Cause then when you append the sql you have a sane name to use instead of needing to have -1 in two spots in the file

Reply:
I agree @SamSaffron, this is very very ugly but I'm simply accessing the hash by id in `def find()` and in def `save_default_query()`, depending on the first key to be the id. I need to be able to search the queries to be able to run it so I'm finding by key here.